css/style.css

```diff
@@ -0,0 +1,493 @@
+body{
+    font-size: 0.8rem;
+    font-family: 'Roboto', Arial, sans-serif;
+}
+
+hr {
+    border: 1px solid #e3e3e3;
+}
+#nav-bar {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    height: 36px;
+    margin: 20px;
+    }
+    
+    #col1{
```

Comment:
- [ ] For good coding practices, avoid the use of id's, since they are directly specific and can break your code, instead, use classes. More in [the article pointed in Pathwright about good and bad coding practices](https://speckyboy.com/good-bad-css-practices/)
![image](https://user-images.githubusercontent.com/53256139/90602808-f003b780-e1bf-11ea-9334-4c9d0f2fa517.png)